racket-common.el

```diff
@@ -87,7 +87,7 @@
     ;; char syntax around the "body" so it's treated as a string for
     ;; indent, nav, font-lock. Think of the \n in #<<ID\n as the open
     ;; | quote and the \n in ^ID\n as the close | quote.
-    ((rx "#<<" (group (+? (not (any blank ?\n)))) (group ?\n))
+    ((rx "#<<" (group (+? (not ?\n))) (group ?\n))
```

Comment:
This `not` syntax doesn't work on some older versions of Emacs that Racket Mode still supports, which is why the automated tests are failing for those.

Instead of `(not ?\n)` I think you need `(not (any ?\n))`.